### DISCUSSION TITLE
Badly overbound oxide with skprogs SK files

### DISCUSSION THREAD

**Discussion by andrewshyichuk** [dftbplus/dftbplus#1557] [Q&A] [answered]
I'm having a very basic problem, seemingly. 

I want to run some calculations for Lu2O3. So, I've used skprogs to make SK files, and succeeded - in the sense that I got myself some brand new SK files. Here's skdef:
[skdef.hsd.txt](https://github.com/user-attachments/files/17497172/skdef.hsd.txt)
 
With those, I've tried to EOS (energy as a function of cell size, the most basic of things). 
And it turned out dramatically bad, it's a flat line, no hint of a minimum. Actually, here it is:
```
0.97                           -4807.0483
0.98                           -4802.7204
0.99                           -4798.3808
1.00                           -4794.0230
1.01                           -4789.6412
1.02                           -4785.2303
1.03                           -4780.7856
```

What am I missing here? It's clearly badly overbound. Should I create SK files for Lu3+ and O2-, other than the default neutral atoms? Albeit, from Lowding charge analysis, Lu is about +0.7, and O is about -0.5, not that much ionic really. 
 
Should I maybe use a softer confinement potential for O? The one I use if from mio set, and looks oddly tight. But then, I've tested it - nope, still bad. 
Here's most recent skdef:
[
[skdef.hsd.txt](https://github.com/user-attachments/files/17506086/skdef.hsd.txt)
](url)

The 4f orbitals of Lu are not the culprit, the twocnt grid is not the culprit, confinement potentials are likely not the culprit.

**Comment by aradi** [comment, marked as answer]
Please, make sure, you also add a repulsive contribution to the SK-files. The electronic ones must be compensated by the appropriate repelling repulsive terms. (See the DFTB literature for more details...)

**Reply by andrewshyichuk** [reply]
I've based my skdef on the example from skprogs. Would you please check if it has any repulsive contribution?
If  not - which keywords should I use?
These skfiles required me to use PolynomialRepulsive in DFTP+, I thus assumed they contain the respective contributions. 

EDIT: From what I can gather by grepping in skprogs py files - there is no repulsive contribution whatsoever. That would explain a lot. Is there a program for that in the dftb+ family?
